cristin-project/src/main/java/no/unit/nva/cristin/projects/common/QueryBuilderProject.java

```diff
@@ -96,58 +90,26 @@ protected void setPath(String key, String value) {
     protected void setValue(String key, String value) {
         var qpKey = keyFromString(key,value);
         switch (qpKey) {
-            case PATH_IDENTITY:
-            case PATH_PROJECT:
-                withPathIdentity(value);
-                break;
-            case PATH_ORGANISATION:
-                withPathOrganization(value);
-                break;
-            case ORGANIZATION:
-                withOrganization(value);
-                break;
-            case BIOBANK:
-                withBiobank(value);
-                break;
-            case KEYWORD:
-                withKeyword(value);
-                break;
-            case LANGUAGE:
-                logger.info("Ignoring language parameter -> " + value);
-                break;
-            case PARTICIPANT:
-                withParticipant(value);
-                break;
-            case QUERY:
-                withQuery(value);
-                break;
-            case STATUS:
-                withStatus(value);
-                break;
-            case APPROVAL_REFERENCE_ID:
-            case APPROVED_BY:
-            case FUNDING:
-            case FUNDING_SOURCE:
-            case GRANT_ID:
-            case INSTITUTION:
-            case LEVELS:
-            case MODIFIED_SINCE:
-            case NAME:
-            case PROJECT_MANAGER:
-            case PROJECT_UNIT:
-            case TITLE:
-            case USER:
-            case PAGE_CURRENT:
-            case PAGE_ITEMS_PER_PAGE:
-            case PAGE_SORT:
+            case PATH_IDENTITY, PATH_PROJECT -> withPathIdentity(value);
```

Comment:
`PATH_IDENTITY `was not set to this before?

Reply:
It was set before as well...
```
  case PATH_IDENTITY:
  case PATH_PROJECT:
           withPathIdentity(value);
```

without `break;` it will just fall through to the next statement , these statements are equal, but java 17 syntax in more explicit.